ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -286,11 +301,22 @@ static bool sqlite_handle_rollback(pdo_dbh_t *dbh)
 
 static int pdo_sqlite_get_attribute(pdo_dbh_t *dbh, zend_long attr, zval *return_value)
 {
+	pdo_sqlite_db_handle *H = (pdo_sqlite_db_handle *)dbh->driver_data;
+
 	switch (attr) {
 		case PDO_ATTR_CLIENT_VERSION:
 		case PDO_ATTR_SERVER_VERSION:
 			ZVAL_STRING(return_value, (char *)sqlite3_libversion());
 			break;
+		case PDO_SQLITE_ATTR_TRANSACTION_MODE:
+			zend_long mode;
+			if (H->transaction_mode == 0) {
+				mode = PDO_SQLITE_ATTR_TRANSACTION_MODE_DEFERRED;
+			} else {
+				mode = H->transaction_mode;
+			}
+			ZVAL_LONG(return_value, mode);
+			break;
```

Comment:
Same as my previous review https://github.com/php/php-src/pull/19317/files#r2243162264, this depends on `driver_data` being zero-initialized.